plaidml/keras/backend_test.py

```diff
@@ -1802,6 +1802,217 @@ def resnetLayer5(self, b, x, k):
         o = b.relu(c)
         return [o]
 
+
```

Comment:
I'm not sure this should go into backend_test.py, backend test is already pretty slow. Also, these comments don't seem generally useful.

Reply:
The tests runs are 14-16 ms each, so it should not take long at all. Removed the comments. They were useful to me to tell me what the constants are in respect to the XSMM calls and the resnet50 layers. Taken out the tests.